lte/gateway/c/session_manager/SessionState.cpp

```diff
@@ -235,26 +235,64 @@ uint32_t SessionState::get_current_version() {
 }
 
 void SessionState::set_current_version(
-    int new_session_version, SessionStateUpdateCriteria* session_uc) {
```

Comment:
Let's keep the `SessionStateUpdateCriteria* session_uc` pointer type here (output parameter that can be optional should always be pointers so it's clear from the signature)

Reply:
Yes, changed as a pointer type.